src/test/java/Homework18.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+ @Test
+    public void playSong() throws InterruptedException {
+
+  provideEmail("valeriya.trygubova@testpro.io");
+  providePassword("ltZaqmXZ");
+  clickSubmit();
+
+  playNextSong();
+  clickPlay();
+
+  //Assert
+  Assert.assertTrue(isSongPlaying());
+
+ }
```

Comment:
HINT

**Use Actions**

![18](https://github.com/trygubov/QA-Automation-Framework-Template/assets/104761482/9c60961f-8d33-4292-ba49-07d8c5f26e1a)

or

if your problem is still not solved, you need a DOM-level solution. So, we need an interface that helps to execute JavaScript with Selenium WebDriver. It is called **JavaScriptExecutor**. JavaScript works at the DOM level. Therefore, we are able to click any web element by using JavaScript executor.

**Clicking Elements in Selenium using JavaScript** (https://www.baeldung.com/java-selenium-javascript)

![18_1](https://github.com/trygubov/QA-Automation-Framework-Template/assets/104761482/d646897d-8d13-4526-b2fa-a44603995b9e)

AND

1) Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

2)    **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
For example

![Clean_Test](https://github.com/trygubov/QA-Automation-Framework-Template/assets/104761482/a82d3d3b-a55b-4e51-a9e5-7acda1a85b7b)